content/departments/cloud/technical-docs/private-code-hosts/index.md

```diff
@@ -134,22 +134,29 @@ i.e. `private.CUSTOMER.com` -> `public.CUSTOMER.com` where:
 
 1. Configure config.yaml
 
+> Important: Follow playbook for given cloud instance to introduce additional node pool, as changing oauth scope on existing one will cause downtime.
```

Comment:
The rendered style doesn't look important at all 😂 
so https://github.com/sourcegraph/handbook/pull/7738